xbmc/cores/VideoPlayer/DVDSubtitles/webvtt/WebVTTHandler.h

```diff
@@ -141,6 +141,11 @@ class CWebVTTHandler
   */
   void DecodeLine(std::string line, std::vector<subtitleData>* subList);
 
+  /*
+   * \brief Return true if the margins are handled by the parser.
+   */
+  bool IsForcedMargins() { return m_overridePositions; };
```

Comment:
this method can be `const` - not a blocker

Reply:
done

Reply:
re-done :)